keps/sig-apps/1591-daemonset-surge/README.md

```diff
@@ -141,6 +145,28 @@ pod). If the old pod delays deletion, then the new pod has a chance to accept ha
   * Testing should set up conflicting rules like HostPort and verify that surge fails and the correct daemonset condition is set and events are generated.
 	* A test should cover a pod going unready during rollout and verifying it is immediately replaced.
 
+
+#### Prerequisite testing updates
+[x] I/we understand the owners of the involved components may require updates to existing tests to make this code solid enough prior to committing the changes necessary to implement this enhancement.
+
+##### Unit tests
+
+
+`k8s.io/kubernetes/pkg/apis/apps/validation`                     `06/06/2022`:     `90.6% of statements`
+`k8s.io/kubernetes/pkg/apis/apps/validation/validation.go:387`:	 `06/06/2022`:		`100.0% of statements`
+`k8s.io/kubernetes/pkg/controller/daemon`: `06/06/2022`: `70.7% of statements`
+`k8s.io/kubernetes/pkg/registry/apps/daemonset`: `06/06/2022`: `31.1% of statements`
+`k8s.io/kubernetes/pkg/registry/apps/daemonset/strategy.go:129`:	`06/06/2022`:	`100.0% of statements`
```

Comment:
Nit: this renders poorly if you switch to rich preview, I'd suggest wrapping this entire block in a code. Also a few words of description per each package would be nice :wink: You and I know what each package has and what it tests, but a sentence per each pacakge/set of packages would be nice. Also add information if you're planning to add any additional tests or the current set is sufficient.

Reply:
I added a statement against against each package which talks about what functionality is being tested as you suggested. PTAL